README.md

```diff
@@ -1,48 +1,64 @@
 # radiance-folder-structure
 
-Folder structure for daylight studies with Radiance.
+Folder structure for lighting studies with Radiance.
 
 ## GOAL
 
-The ultimate goal of this project is to provide an input data folder / file structure
-that can be used both by Radiance users and software developers. Using an standard folder
-structure will makes it easy to use different tools / scripts regardless of the source of
-the files.
+The ultimate goal of this project is to provide an example for standard Radiance input
+data folder/file structure that can be used both by Radiance users and software
+developers.
 
-See README.md inside each folder for more information.
+## Folder structure
+
+The Radiance folder has a single required sub-folder for `model`. A `model` carries all
+the information about the geometry, modifiers and describes dynamic parts of the model if
+any. All the data in a `model` folder are reusable between different types of studies.
+That's why it does <u>NOT</u> include files such as `sky` which are specific to a certain
+type of study AKA recipe. This information should be included in the `recipe` folder
```

Comment:
This makes it sound like `recipe` is the official name of the folder instead of it bearing the name of the type of study.